railties/lib/rails/generators/rails/app/templates/config/storage.yml.tt

```diff
@@ -18,7 +18,7 @@ local:
 # google:
 #   service: GCS
 #   project: your_project
-#   keyfile: <%%= Rails.root.join("path/to/gcs.keyfile") %>
+#   credentials: <%%= Rails.root.join("path/to/gcs.keyfile") %>
```

Comment:
Is the keyfile something users should best keep out of their repo? Then we could recommend `Rails.application.encrypted("config/gcs.credentials")` so they could keep it encrypted in repo.

Reply:
The keyfile is JSON format (Google's recommended format). However, `encrypted` is supposed to use the yaml format. Therefore, I think that it is difficult to use `encrypted` as it is.

Reply:
Oh wait, this isn't the same thing. `credentials:` takes a file path so it shouldn't be encrypted, I think.

Reply:
It also takes a Hash.

Reply:
I don’t think `Rails.application.encrypted` requires that the encrypted file have any particular format, so I do think we should prefer it here.

Reply:
Scratch that. We’d still need to decode the JSON contents of the file.